atlasdb-config/src/test/java/com/palantir/atlasdb/factory/AutoServiceAnnotatedAtlasDbFactory.java

```diff
@@ -50,7 +50,7 @@ public KeyValueService createRawKeyValueService(KeyValueServiceConfig config, Op
     }
 
     @Override
-    public TimestampService createTimestampService(KeyValueService rawKvs) {
+    public TimestampService createTimestampService(KeyValueService rawKvs, boolean initializeAsync) {
```

Comment:
should perhaps javadoc or warn that `initializeAsync` is ignored.

Reply:
Done.

Reply:
Not from where I'm looking!